internal/pkg/cli/task_run.go

```diff
@@ -110,6 +122,17 @@ func (o *runTaskOpts) Validate() error {
 	return nil
 }
 
+// Ask prompts the user for any required or important fields that are not provided.
+func (o *runTaskOpts) Ask() error {
+	if err := o.askTaskFamilyName(); err != nil {
+		return err
+	}
+	if err := o.askEnvName(); err != nil {
+		return err
+	}
```

Comment:
Should we add an ask method for the number of tasks to run?

Reply:
That would basically make "number of tasks" a required field right? But it is (currently meant to be) an optional field which default value to 1.

Reply:
We can have default value for the prompt but +1 on that, since I think for most of cases the num will just be 1.